tools/provenance-generator/kn-attestation.json

```diff
@@ -0,0 +1,56 @@
+{
+    "_type": "https://in-toto.io/Statement/v0.1",
+    "predicateType": "https://slsa.dev/provenance/v0.2",
+    "subject": [
+      {
+        "name": "gcr.io/knative-releases/knative.dev/serving/cmd/controller",
+        "digest": {
+          "sha256": "bac158dfb0c73d13ed42266ba287f1a86192c0ba581e23fbe012d30a1c34837c"
+        }
+      },
+      {
+        "name": "gcr.io/knative-releases/knative.dev/serving/cmd/queue",
+        "digest": {
+          "sha256": "83f6888ea9561495f67334d044ffa8ad067d251ad953358dda7ea5183390cc69"
+        }
+      }
+    ],
+    "predicate": {
+      "builder": {
+        "id": "https://prow.knative.dev"
+      },
+      "buildType": "https://cloudbuild.googleapis.com/CloudBuildYaml@v1",
```

Comment:
Is this right?

Reply:
No, I still don't know the right values. Will need to talk to SLSA and puerco about that.

Reply:
In this case we need a URL to let others what kind of build we are running. In general, I think this is most useful when you are running a build-as-code scenario. This would point reviewers to  the build definition spec.

My opinion here is that since prow is running a shell script, it is probably self-explanatory and we can point the URL to the shell script.